anvil-python/anvil/commands/utils.py

```diff
@@ -49,3 +59,148 @@ def juju_login(ctx: click.Context) -> None:
+            ctx,
+        )
+    return value
+
+
+@click.command(
+    cls=FormatEpilogCommand,
+    epilog="""
+    \b
+    Create a MAAS admin account with the following details:
+    Username: admin
+    Password: VerySecure9000
+    Email: admin@company.com
+    Launchpad account to import SSH key from: lp-username
+    \b
+    maas-anvil create-admin --username admin --password VerySecure9000 --email admin@company.com --ssh-import lp:lp-username""",
+)
+@click.option(
+    "--username",
+    help="The username for the new admin account",
+    required=True,
+)
+@click.option(
+    "--password",
+    help="The password for the new admin account",
+    required=True,
+)
+@click.option(
+    "--email",
+    help="The email address for the new admin account",
+    required=True,
+)
+@click.option(
+    "--ssh-import",
+    help="Import SSH keys from Launchpad (lp:user-id) or GitHub (gh:user-id)",
+    callback=validate_ssh_import,
+)
+def create_admin(
+    username: str,
+    password: str,
+    email: str,
+    ssh_import: str | None,
+) -> None:
+    """Creates a MAAS admin account."""
+    cmd = [
+        "juju",
+        "run",
+        "maas-region/0",
+        "create-admin",
+        "--format=json",
+        f"username={username}",
+        f"password={password}",
+        f"email={email}",
+    ]
+    if ssh_import:
+        cmd.append(f"ssh-import={ssh_import}")
+    try:
+        result = subprocess.run(cmd, check=True, capture_output=True)
+    except subprocess.CalledProcessError as e:
+        raise RuntimeError(
+            f"Failed to create MAAS admin account: {e.stderr.decode()}"
+        )
+    try:
+        result_dict = json.loads(result.stdout.decode())
+        if result_dict["maas-region/0"]["status"] == "failed":
+            raise RuntimeError(
+                "Failed to create admin account, response from maas-region: "
+                f"{result_dict['maas-region/0']['message']}: "
+                f"{result_dict['maas-region/0']['results']['stderr']}"
+            )
+    except (KeyError, json.JSONDecodeError):
+        LOG.error(
+            f"Unknown response from maas-region/0 when getting API key: {result.stdout.decode()}"
+        )
+        raise RuntimeError("Failed to parse response from maas-region/0")
+    console.print("MAAS admin account has been successfully created.")
+
+
+@click.command(
+    cls=FormatEpilogCommand,
+    epilog="""
+    \b
+    Get a MAAS API key for the admin account:
+    maas-anvil get-api-key --username admin""",
+)
+@click.option(
+    "--username",
+    help="The username to retrieve an API key for",
+    required=True,
+)
+@click.option(
+    "-f",
+    "--format",
+    type=click.Choice([FORMAT_DEFAULT, FORMAT_VALUE, FORMAT_YAML]),
+    default=FORMAT_DEFAULT,
+    help="Output format of the API key.",
+)
+def get_api_key(
+    username: str,
+    format: str,
+) -> None:
+    """Retrieves an API key for MAAS"""
```

Comment:
nit: let's add a period in the end
```suggestion
    """Retrieves an API key for MAAS."""
```